docs/AccessControl.md

```diff
@@ -113,6 +113,246 @@ This specifies that the file is expected to be transmitted from the caller to
+`stream`, `file` and `auth`, an application may extend this list, and
+also it's not obliged to support all of the standard ones.
+
+
+### SRT_REJX_LOCKED
+
+The resource being accessed is locked for any access. This is similar to
+`SRT_REJX_CONFLICT`, but in this case it's locked for reading and writing.
+This is for a case when the resource should be shown as existing and in
+general available for the client, just temporarily blocked for access at
+the moment.
+
+
+### SRT_REJX_FAILED_DEPEND 
+
+The dependent entity for the request is not present. In this case the
+dependent entity is the session, which should be specified in the `s`
+key. This means that the specified session ID is nonexistent or it
+has already expired.
+
+
+### SRT_REJX_ISE
+
+Internal server error. This is for a general case when the request has
+been correctly verified and no problems related to this have been found,
+the processing of the request has started and the activity was about to
+be correctly performed, just there was some unexpected error during this
+process.
+
+
+### SRT_REJX_UNIMPLEMENTED
+
+The request was correctly recognized, but the current version doesn't
+support it. This should be reported for a case, when some features to
+be specified in the StreamID request are supposed to be supported in a
+predictable feature, but the current version of the server does not
+support it, or the support for this feature in this version has been
+temporarily blocked. This shouldn't be reported in case of deprecated
+features or features that were decided as no longer supported.
+
+
+### SRT_REJX_GW
+
+The server acts as a gateway and the target endpoint rejected the
+connection. It is unspecified by what exactly reason the connection
+was rejected. The gateway shall not forward the original rejection
+code from the target endpoint because this would suggest the error
+on the gateway itself - rather then this error, with possibly some
+other way to report the original target error, if there is a possibility.
+
+
+### SRT_REJX_DOWN
+
+The service is down for maintenance. This can only be reported in
+case when the servis has been temporarily replaced by a stub that is only
+reporting this error, while the real service is down for maintenance.
+
+
+### SRT_REJX_VERSION
+
+Application version not supported. This can refer to both not supported
+application feature - also due to too low SRT version - or about referring
+to a feature that is no longer supported as a cut off backward comptibility.
+
+
+### SRT_REJX_NOROOM
+
+The data stream cannot be archived due to lacking storage space. This is
+reported in case when the request type was to send a file or a live stream
+to be archived. Note that a file transmission is usually pre-declaring the
+file lentgh, so this error can be reported early, but it can be as well
+reported when the stream is of undefined length and there's no more storage
+space made available for this stream.
```

Comment:
```suggestion
The data stream cannot be archived due to a lack of storage space. This is
reported a request to send a file or a live stream to be archived is unsuccessful. 
Note that the length of a file transmission is usually pre-declared, so this error 
can be reported early. It can also be reported when the stream is of undefined 
length, and there is no more storage space available.
```

Reply:
I think there's lacking one word here.

`This is reported` **when** `a request to send a file or a live stream to be archived is unsuccessful` ?

Reply:
Yes, I missed that.